src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -165,37 +159,60 @@ public ProcessResult updateCase(
         } catch (UnprocessableEntity exception) {
             ClientServiceErrorResponse errorResponse = serviceResponseParser.parseResponseBody(exception);
             return new ProcessResult(errorResponse.warnings, errorResponse.errors);
+        } catch (FeignException.UnprocessableEntity exception) {
+            String msg = getErrorMessage(configItem.getService(), existingCaseId, exceptionRecord.id)
+                + " - CCD could not process request";
+            // is there a better way?
+            boolean isStartEvent = exception.request().url().contains("event-triggers");
+
+            log.error(msg, exception);
+
+            if (!isStartEvent) {
+                return new ProcessResult(singletonList(msg), emptyList());
+            } else {
+                throw new CallbackException(msg, exception);
+            }
```

Comment:
Why is there a difference here?
Perhaps worth leaving a comment in code.

Reply:
ok, i can add a comment. smth like:

> difference between previous exception handling of start and submit events

i'll think of more _elaborative_ message